### ISSUE TITLE
VoLTE will crash

### ISSUE THREAD

**Issue by YuZebra** [pjsip/pjproject#3676]
### Describe the bug

When I use the 5G network of my mobile phone, if VoLTE is supported, just calling pjsua_acc_add will crash and get stuck.

### Steps to reproduce

This is my code

var acc_cfg = pjsua_acc_config()
pjsua_acc_config_default(&acc_cfg)
acc_cfg.id = pj_str(id)
acc_cfg.cred_count = 1
acc_cfg.cred_info.0.username = pj_str(username)
acc_cfg.cred_info.0.realm = pj_str(realm)
acc_cfg.cred_info.0.data = pj_str(passwd)
acc_cfg.reg_uri = pj_str(registrar)
acc_cfg.proxy_cnt = 1
acc_cfg.proxy.0 = pj_str(proxy)
acc_cfg.vid_out_auto_transmit = pj_bool_t(PJ_TRUE.rawValue)
acc_cfg.vid_in_auto_show = pj_bool_t(PJ_TRUE.rawValue)
          
/* Add account */
pjsua_acc_add(&acc_cfg, pj_bool_t(PJ_TRUE.rawValue), nil)

It will crash

### PJSIP version

2.12

### Context

iPhone 

### Log, call stack, etc

```shell
Assertion failed: (pjsua_var.tpdata[0].data.ptr != ((void*)0)), function pjsua_acc_add, file pjsua_acc.c, line 441.
```

**Comment by sauwming** [comment]
You must create at least one transport:
https://github.com/pjsip/pjproject/blob/master/pjsip/src/pjsua-lib/pjsua_acc.c#L465

**Comment by YuZebra** [comment]
> You must create at least one transport: https://github.com/pjsip/pjproject/blob/master/pjsip/src/pjsua-lib/pjsua_acc.c#L465

Yes, when I saw this error, I also suspected that there were too few settings
But I still can't figure out which setting is missing (I'm not really familiar with everything about VOIP)

Here is my code
var transport_id = pjsua_transport_id()
var tcp_cfg = pjsua_transport_config()
pjsua_transport_config_default(&tcp_cfg)
tcp_cfg.port = 5060
status = pjsua_transport_create(PJSIP_TRANSPORT_UDP, &tcp_cfg, &transport_id)

I saw in tcp_cfg that public_addr has ptr, and another bound_addr also has it. As for the content of ptr, I am not familiar with what kind of content should be filled in.

Most of the teaching examples related to the Internet only mention UnsafeMutablePointer<Int8>(mutating: ("0.0.0.0" as NSString).utf8String)

So I added
tcp_cfg.public_addr.ptr = UnsafeMutablePointer<Int8>(mutating: ("my sip server ip" as NSString).utf8String)

Also, I have var acc_cfg = pjsua_acc_config()
The cred_info.0.data.ptr here also sees the existence of ptr

No matter what content I filled in, it was still an error. I felt hopeless during the development process because the previous version would not cause the program to crash because I missed setting this thing.

**Comment by sauwming** [comment]
I would suggest to test with the sample app and try to reproduce the issue there.

**Comment by YuZebra** [comment]
I reused the sample code for testing, and then discovered that it might be a problem with Taiwan's telecom operators.

When I use the network of telecom provider A, I can connect normally to the SIP Server I want to connect to, but if I switch to another telecom provider B, it will crash.

This is true whether I use TCP or UDP, but as long as I change another Port, there will be no such problem. Once I use Port 5060, it will happen.

During the connection process, I also found that when I use 5066, the port in the connection within TCP and the message printed by SIP can be seen to change dynamically.
